<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMet</class>
 <widget class="QWidget" name="WidgetMet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxMet">
       <property name="text">
        <string>activar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>379</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteDataMet">
       <property name="text">
        <string>Delete data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelOutFolderMet">
       <property name="text">
        <string>Outfolder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMet">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonMet">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelAmbitoProyectoMet">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Zone project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMet">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="labelTamanyoPixelMet">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Pixel size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTamanyoPixelMet"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="labelCoordinateSystemMet">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Coordinate system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCoordinateSystemMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxCortarMet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Cut DTM</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="labelCortarMet">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Size cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxCortarMet"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="labelNumeroCanalesspasadaMet">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Numbers chanels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxNumeroCanalesspasadaMet"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxFootPrintMaskMet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>FootPrintMask</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="labelSelectSensor">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Select sensor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSelectSensor">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="labelOffsetPasadaMet">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Offset flight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxOffsetPasadaMet"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="labelAnchoPasadaMet">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>width flight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxAnchoPasadaMet"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
